test/performance/run.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -o xtrace
+set -o errexit
+
+kubectl apply -f - <<EOF
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: KubeArchiveConfig
+metadata:
+  name: kubearchive
+  namespace: default
+spec:
+  resources:
+    - selector:
+        apiVersion: v1
+        kind: Pod
+      archiveWhen: status.phase == "Succeeded"
+EOF
+
+kubectl create clusterrole view-pods --verb=list,get --resource=pods,pods/status --dry-run=client -o yaml | kubectl apply -f -
+kubectl create clusterrolebinding view-pods --clusterrole view-pods --user system:serviceaccount:default:default --dry-run=client -o yaml | kubectl apply -f -
+export SA_TOKEN=$(kubectl create token default --namespace default)
+
+trap 'kill $(jobs -p)' EXIT
+
+python3 -m venv venv
+./venv/bin/python -m pip install locust cloudevents
```

Comment:
I like having everything set up in the script but I would suggest to have the dependencies also in a requirements.txt (even though it will be a small file with just two lines: `locust` and `cloudevents`) and replace then this line for:

```suggestion
./venv/bin/python -m pip install -r requirements.txt
```

This way, you don't need to run the script to have the dependencies in place and it's also a little awkward to not see a `Pipfile` or a `requirements.txt` when dealing with python code.

Also, some IDEs automate the creation of venv with requirements if a `requirements.txt` or a `Pipfile` is found, so it would be nice for those usecases.

Reply:
Done